src/Compilers/Core/Portable/Operations/ArgumentKind.cs

```diff
@@ -12,21 +10,20 @@ public enum ArgumentKind
         None = 0x0,
 
         /// <summary>
-        /// Argument is specified positionally and matches the parameter of the same ordinality.
-        /// </summary>
-        Positional = 0x1,
-        /// <summary>
-        /// Argument is specified by name and matches the parameter of the same name.
+        /// Argument value is explicitly supplied.
         /// </summary>
-        Named = 0x2,
+        Explicit = 0x1,
+
         /// <summary>
-        /// Argument becomes an element of an array that matches a trailing C# params or VB ParamArray parameter.
+        /// Argument is a param array created by compilers for the matching C# params or VB ParamArray parameter. 
+        /// Note, the value is a an array creation expression that encapsulates all the elements, if any.
         /// </summary>
-        ParamArray = 0x3,
+        ParamArray = 0x2,
+
         /// <summary>
-        /// Argument was omitted in source but has a default value supplied automatically.
+        /// Argument is a default value supplied automatically by the compilers.
         /// </summary>
-        DefaultValue = 0x4
+        DefaultValue = 0x3
```

Comment:
is this redundant with Explicit?  If you don't explicitly provide an argument, won't you get a default value?